cmd/cluster/health.go

```diff
@@ -221,18 +261,24 @@ func ocmDescribe(clusterID string) ([]string, string, int, error) {
 	cluster := response.Body()
 	cloudProvider := cluster.CloudProvider().ID()
 	cloudProviderMessage := strings.ToUpper(cloudProvider)
+	clusterInfo.Name = cluster.Name()
 
 	healthObject.ID = cluster.ID()
 	healthObject.Name = cluster.Name()
 	healthObject.Provider = cloudProviderMessage
+	healthObject.AZs = cluster.Nodes().AvailabilityZones()
+	healthObject.Expected.Infra = cluster.Nodes().Infra()
+	healthObject.Expected.Master = cluster.Nodes().Master()
 
```

Comment:
Maybe I'm missing this but it's not clear to me where this is being used within this function.  Is this some kind of globally scoped variable `healthObject`?

Reply:
Ahh, I see where this is coming from now.

I think we should take _out_ the struct fields from the `healthOptions` struct, and load all of them into the `clusterInfo` struct.  The `healthOptions` struct is more for options being passed into the command from the end-user, and I don't think overloading that struct and storing all of the various data about a cluster in there is the best choice when we can just push all of that data into the `clusterInfo` struct instead.